watchtower/wtdb/client_db.go

```diff
@@ -1031,44 +1074,100 @@ func getClientSessionBody(sessions kvdb.RBucket,
 	return &session, nil
 }
 
+// PerAckedUpdateCB describes the signature of a callback function that can
+// be called for each of a session's acked updates.
+type PerAckedUpdateCB func(*ClientSession, uint16, BackupID)
+
+// PerCommittedUpdateCB describes the signature of a callback function that
+// can be called for each of a session's committed updates (updates that the
+// client has not yet received an ACK for).
+type PerCommittedUpdateCB func(*ClientSession, *CommittedUpdate)
+
+// ClientSessionListOption describes the signature of a functional option of
```

Comment:
nit: 
> ..of a functional option of that..

typo?

Reply:
yes a typo. Will fix it in next update after a full review :) gonna re-request review in the mean time

Reply:
fixed :)